### ISSUE TITLE
Web UX: Enable selectable text 

### ISSUE THREAD

**Issue by esDotDev** [gskinnerTeam/flutter-wonderous-app#132]
Text should be selectable on web and desktop platforms (maybe all platforms?).

Ideally it can be copy/pasted as well. 

Note: This will require having different scroll behaviors on the editorial view, as "drag to scroll" will need to be disabled if we want to select text over multiple lines.

**Comment by AlexGarneau** [comment]
Notes from Shawn:

**
The intent here is mostly to feel more web-like, we don’t really care if they can copy the text as much as it feels as close as possible to a native web app.

We probably wouldn't modify the package but we might use a web specific alternative treatment that doesnt break text selection. might be an acceptable trade off, would want to get Jared's thoughts.

IE: maintain true drop caps on mobile, do something drop cap-like on web that doesn't break text selection. (edited) 

Another option could be to put like a floating Copy btn on this page specifically , similar to GPT, with the excuse that its draggable and we want to maintain that. Then on other screens we enable text selection where it makes sense.
**

TLDR: This issue is less about the text being copy/pastable and more that it more closely resembles a normal HTML page.
